### ISSUE TITLE
findBreakingChanges does not support extended types

### ISSUE THREAD

**Issue by erangakm** [graphql/graphql-js#2055]
Old type:
```
extend type Report {
  displayName: String!
}

type Report {
  id: Int!
}
```

New type:
```
extend type Report {
  displayName: String
}

type Report {
  id: Int!
}
```

Removing the nullability on the extended type doesn't trigger a breaking change at the moment. Would be great if this is supported :)

**Comment by IvanGoncharov** [comment]
@erangakm What function you use to build your schemas?
Is it `buildSchema`?
Problem is that `buidlSchema` simply ignores extensions node as discussed in #922.

**Comment by erangakm** [comment]
@IvanGoncharov I'm using ``makeExecutableSchema``

**Comment by IvanGoncharov** [comment]
@erangakm Thanks for additional info I will try to take a look.
You can help a lot if you create a small repo that reproduces this issue.

**Comment by yaacovCR** [comment]
Closed by #2248
